staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -101,6 +105,10 @@ func doCleanupMountPoint(mountPath string, mounter Interface, extensiveMountPoin
 		return err
 	}
 
+	if mounter.CanSafelySkipMountPointCheck() {
```

Comment:
if mountPath is not a mount point, will it return error at line 104 during Unmount? In this case, it will not call removePath in the following, right?

Reply:
It will not return an error. The contract of `CanSafelySkipMountPointChecks` is that `Unmount` does not produce such errors: https://github.com/kubernetes/kubernetes/blob/7e674280561f4bbc34f48a3fda2281d2978fbf36/staging/src/k8s.io/mount-utils/mount.go#L69

The linux impl ignores this error here, if it has detected the 'safe' behavior: https://github.com/kubernetes/kubernetes/blob/7e674280561f4bbc34f48a3fda2281d2978fbf36/staging/src/k8s.io/mount-utils/mount_linux.go#L345

Reply:
Q: Why do we need this call here? Since `mounter.withSafeNotMountedBehavior && strings.Contains(string(output), errNotMounted)` already hides the error. can you not ignore this check?